ui/raidboss/data/04-sb/dungeon/the_burn.js

```diff
@@ -227,10 +236,10 @@
         'The Scorpion\'s Den': 'Nid du scorpion',
         'The Gamma Segregate': 'Ancien répartiteur gamma',
         'The Aspersory': 'Plateau des Brumes',
-        'Rock Biter': 'trancheur de pierre',
-        'Mist Dragon': 'dragon des brumes',
-        'Dim Crystal': 'cristal décoloré',
-        'Defective Drone': 'drone défectueux',
+        'Rock Biter': 'Trancheur de pierre',
```

Comment:
For what it's worth, there's no reason to change capitalization of syncs.  They are all case insensitive.  I think in general it's not worth making these changes.

Reply:
Yeah I know, it's just ugly and hard to read.